src/app/modules/settings/components/settings-integrations/settings-integrations.component.scss

```diff
@@ -3,6 +3,46 @@
   font-weight: 600;
 }
 
+.onboarding-titles {
+  font-size: 1.16667rem;
+  font-style: normal;
+  font-weight: 600;
```

Comment:
From what I remember, you could user CSS "inliner" from the HTML to specify font-weight, font-style, and even colors (for colors only if they are specified in the global bootstrap variable I think). You should check with @Hugonotfound to make this cleaner

Reply:
Don't spend too long on this. More than 5min => check with Hugo since he will know by heart the inliners 👍